singleton.py

```diff
@@ -0,0 +1,9 @@
+class Singleton:  
+    def __new__(cls):
+        if not hasattr(cls, 'instance'):
+           cls.instance = super(Singleton, cls).__new__(cls)
+        return cls.instance     
+
+
+sing=Singleton() #<__main__.Singleton object at 0x03D3D810>
```

Comment:
PEP 8

Reply:
Let's skip it for this time because of another tasks